src/main.rs

```diff
@@ -132,7 +152,16 @@ fn validate_ecosystems(ecosystem_map: &EcosystemMap) -> Vec<ValidationError> {
             }
         }
 
+        if let Some(ref orgs) = ecosystem.github_organizations {
```

Comment:
This can be cleaner and avoiding a mutable variable with:
```
let has_orgs = ecosystem.github_organizations
    .as_ref()
    .map_or(false, |orgs| !orgs.is_empty());
```

Reply:
Great catch - this looks much better. Updated!